docs/guides/dataweavetransformation.md

```diff
@@ -0,0 +1,306 @@
+    
+```cmd
+kubectl exec -ti curl -- curl -v "http://broker-ingress.knative-eventing.svc.cluster.local/default/demo" \
+  -H "Ce-Specversion: 1.0" \
+  -H "Ce-Type: xml.document" \
+  -H "Ce-Source: curl.shell" \
+  -H "Content-Type: application/xml" \
+  -H "Ce-Id: 1234-abcd" \
+  -d '<?xml version="1.0"?>
+<order><product><price>5</price><model>Company 2020</model></product><item_amount>3</item_amount><payment><payment-type>credit-card</payment-type><currency>USD</currency><installments>1</installments></payment><buyer><email>james@hotmail.com</email><name>James</name><address>Cocodrile Boulevard 61</address><city>Seattle</city><state>CA</state><postCode>98101</postCode><nationality>USA</nationality></buyer><shop>main branch</shop><salesperson>Liam Smith</salesperson></order>'
+```
+
+
+### Viewing the Transformation's Output in the Event Display
+With our event flow in place, we can now view the transformed data in the `EventDisplay`.
+
+We need to retrieve the `EventDisplay` Pod name by running the following command:
+
+```cmd
+kubectl get pods                                                          
+NAME                                                   READY   STATUS    RESTARTS   AGE
+curl                                                   1/1     Running   0          4m36s
+event-display-00001-deployment-fb48c8d7c-g7bmv        2/2     Running   0          3s
+dataweavetransformation-demo-00001-deployment-7978655d45-jsfbr   2/2     Running   0          3s
+```
+With the Pod name, we can run the following command to view the transformed data in the `EventDisplay` Pod logs:
+
+```cmd
+kubectl logs event-display-00001-deployment-fb48c8d7c-g7bmv user-container
+☁️  cloudevents.Event
+Context Attributes,
+  specversion: 1.0
+  type: xml.document
+  source: curl.shell
+  id: 1234-abcd
+  datacontenttype: application/xml
+Extensions,
+  knativearrivaltime: 2022-05-09T10:32:43.32759997Z
+Data,
+  <?xml version="1.0"?>
+<order><product><price>5</price><model>Company 2020</model></product><item_amount>3</item_amount><payment><payment-type>credit-card</payment-type><currency>USD</currency><installments>1</installments></payment><buyer><email>james@hotmail.com</email><name>James</name><address>Cocodrile Boulevard 61</address><city>Seattle</city><state>CA</state><postCode>98101</postCode><nationality>USA</nationality></buyer><shop>main branch</shop><salesperson>Liam Smith</salesperson></order>
+☁️  cloudevents.Event
+Context Attributes,
+  specversion: 1.0
+  type: xml.document.response
+  source: curl.shell
+  id: 1234-abcd
+  time: 2022-05-09T10:32:44.697405628Z
+  datacontenttype: application/json
+Extensions,
+  knativearrivaltime: 2022-05-09T10:32:44.699348327Z
+Data,
+  {
+    "email": "james@hotmail.com",
+    "name": "James"
+  }
+```
+
+We now see the incoming event and the transformed data, as expected. 
+
+
+### Sending the parameters in the request.
+    1. Execute the following command to emit a cloudevent to the broker we created: 
```

Comment:
to add at the beginning: "Now we can try passing parameters such as the DataWeave spell and input and output content types as part of the Cloud Event."